core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -15,3 +15,34 @@
 /**
  * @fileoverview End-to-end tests for user profile features.
  */
+
+var users = require('../protractor_utils/users.js');
+
+var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
+var SubscriptionDashboardPage = require(
+  '../protractor_utils/SubscriptionDashboardPage.js');
+
+describe('Profile Feature', function() {
```

Comment:
Drop `Feature` since it's already assumed that this is a feature :)

Reply:
Done!